adserver/tasks.py

```diff
@@ -417,3 +420,28 @@ def calculate_publisher_ctrs(days=7):
 
         publisher.sampled_ctr = report.total["ctr"]
         publisher.save()
+
+
+@app.task()
+def notify_of_completed_flights():
```

Comment:
Should this check for a `SLACK_TOKEN`?

Reply:
I originally had that and I took it out but maybe we should have it (in the inner loop).